### DISCUSSION TITLE
Translation for git projects

### DISCUSSION THREAD

**Discussion by althafpt** [community/community#3985] [Mobile]
Hey I have noticed some of git projects are in different languages so is there a way in which I can translate it in git

**Comment by zeke** [comment]
Hi @althafpt and @lalap3724 👋🏼 

GitHub for mobile is available in the following languages:

- English
- Japanese
- Brazilian Portuguese
- Simplified Chinese
- Spanish

If you configure the language on your device to a supported language, GitHub for mobile will default to the language. You can change the language for GitHub for mobile in GitHub for mobile's Settings menu.

---

☝🏼 This refers to the user interface of the mobile app itself, not the user-generated content of repos, issues, pull requests, etc, for which there is currently no support.

There is a [new feature for translating user comments in Discussions](https://github.blog/changelog/2021-07-19-machine-translation-for-discussions-content/) on GitHub.com (the website). We're running that as an experiment to see how useful machine translation can be useful for users. We'd love for you to give that a try and share feedback.

Can you share more details about what specifically would be useful for you? Can you also clarify what you mean by "translate it in git"?

Thanks!

**Comment by ousmanmahammedmahie** [comment]
+251926719091

**Comment by ousmanmahammedmahie** [comment]
****
